api/src/main/java/org/openmrs/validator/PersonNameValidator.java

```diff
@@ -164,4 +179,22 @@ private String getFieldKey(String field, boolean arrayInd, boolean testInd) {
 		return testInd ? field : arrayInd ? "names[0]." + field : "name." + field;
 	}
 	
+	/*
+	 * Checks that the given {@link PersonName} field has no trailing and leading spaces.
+	 * @param value the value name.
+	 * @return containsSpace
+	 * @should return true if given field contains trailing or leading space(s), otherwise false.
```

Comment:
remove javadocs of this private method, the method has a clear name which says it all